packages/cozy-harvest-lib/src/connections/accounts.js

```diff
@@ -0,0 +1,87 @@
+const ACCOUNTS_DOCTYPE = 'io.cozy.accounts'
+const PERMISSIONS_DOCTYPE = 'io.cozy.permissions'
+
+export const accountsMutations = client => {
+  const createAccount = async attributes => {
+    const { data } = await client.create(ACCOUNTS_DOCTYPE, attributes)
+    return data
+  }
+
+  const findParentAccount = async id => {
```

Comment:
I would rename this function to `findAccount` since there is nothing related to a parent here I think.